compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -423,8 +425,15 @@ void addPadPackBasedPassPipeline(OpPassManager &funcPassManager,
   addAMDAIEBufferizePasses(funcPassManager);
 }
 
+void addDepthwiseConvolutionDecomposePassPipeline(
+    OpPassManager &funcPassManager, TilingConfig &tilingConfig) {
+  // Currently the pipelines are the same, but in the future they might diverge.
+  addConvDecomposePassPipeline(funcPassManager, tilingConfig);
```

Comment:
High level question, what possible change we can make to depthwise convolution ops but not to regular conv2d ops?  If only the tiling strategy is changed then we shouldn't create a new pipeline.

Reply:
Nothing obvious for now, I'm going to change this to use a single pipeline to be consistent with llvm-cpu, thanks for pointing me to that.